scripts/extractions/extract_meteo.py

```diff
@@ -0,0 +1,162 @@
+"""Extract AGERA5 (Meteo) data using OpenEO-GFMAP package."""
+import argparse
+from functools import partial
+from pathlib import Path
+
+import geojson
+import geopandas as gpd
+import openeo
+import pandas as pd
+from extract_sar import (
```

Comment:
Is this the most elegant way to do this? Importing private methods from other script seems wild. Maybe we should have this integrated as non-private methods in gfmap or have some utilities here that exposes them.